examples/40_advanced/example_text_preprocessing.py

```diff
@@ -100,5 +87,7 @@
 # Get the Score of the final ensemble
 # ===================================
 
+# get predictions for formerly unseen data. Ensure that the data has the same format as the training
```

Comment:
I don't fully understand why this comment is here.

Reply:
i think i overthought this i think it is somehow self explanatory in the way the variables are named